src/ansible_dev_tools/resources/server/creator.py

```diff
@@ -18,6 +18,9 @@
 from ansible_dev_tools.server_utils import validate_request, validate_response
 
 
+MIN_CREATOR_VERSION = "24.10.1"
```

Comment:
We can remove this as we're using `ansible-creator>=24.10.1`

Reply:
Done!